OpenRA.Mods.Cnc/Traits/World/VoxelNormalsPalette.cs

```diff
@@ -59,7 +59,6 @@ public void LoadPalettes(WorldRenderer wr)
 			wr.AddPalette(info.Name, new ImmutablePalette(data));
 		}
 
-		// Normal vector tables from http://www.sleipnirstuff.com/forum/viewtopic.php?t=8048
```

Comment:
https://web.archive.org/web/20041022134721/http://www.sleipnirstuff.com/forum/viewtopic.php?t=8048

Reply:
that's one way to go about it